webapp/components/CommentList/CommentList.spec.js

```diff
@@ -26,6 +25,23 @@ describe('CommentList.vue', () => {
               id: 'comment134',
               contentExcerpt: 'this is a comment',
               content: 'this is a comment',
+              author: {
+                id: 'some-user',
+                slug: 'some-slug',
+              },
+            },
+            {
+              id: 'comment135',
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-08-07T08:30:56Z" title="Friday, August 7th 2020, 10:30:56 am +02:00">Aug 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -28,6 +28,20 @@ describe('CommentList.vue', () => {
               content: 'this is a comment',
               author: { id: 'some-user' },
             },
+            {
+              id: 'comment135',
+              contentExcerpt: 'this is a deleted comment',
+              content: 'this is a deleted comment',
+              deleted: true,
+              author: { id: 'some-user' },
+            },
+            {
+              id: 'comment136',
+              contentExcerpt: 'this is a disabled comment',
+              content: 'this is a disabled comment',
+              disabled: true,
+              author: { id: 'some-user' },
+            },
```


Okay. Cool!
Then this test extension is needed. 😍